HTML/HTML.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+
+<body style="font-family: Vemana2000, sans-serif;">
+<ul style="list-style-type: none; overflow: hidden; background-color: black; margin-left: 0%; padding-top:25px; padding-bottom: 0px;">
+    <li class="li" style="display: block; float: left; margin-right:27px; font-size:20px;">
+        <img src="https://logospng.org/wp-content/uploads/html-5.png" width="70px;">
+    </li>
+    <li class="li" style="display: block; float: left; margin-right:27px; font-size:20px;">
+        <a href="Git.html" style="color:white; text-decoration: none;">Git</a>
+    </li>
+    <li style="display: block; float: left; margin-right:27px; ">
+        <a href="HTML.html" style="color:white; text-decoration: none; font-size:20px; ">HTML</a>
+    </li>
+    <li style="display: block; float: left; margin-right:27px; font-size:20px;">
+        <a href="Javascript.html" style="color:white; text-decoration: none;">Javascript</a>
+    </li>
+    <li style="display: block; float: left; margin-right:27px; font-size:20px;">
+        <a href="Ghibli.html" style="color:white; text-decoration: none;">Studio Ghibli</a>
+    </li>
+</ul>
+<p> HTML steht fur Hypertext Markup Language und ist die Grundlage vom der Webbasierten Textverarbeitung.<br>
+    Sie wurde in CERN entwickelt und kann dabei von so gut wie jedem modernen Browser übersetzt werden.<br> Es basiert auf vielen verschiedenen Elementen die genutzt werden um den Inhalt zu umschließen und diesen damit unterschiedliche Eigenschaften in ihrer Anordnung zu geben. </p>
+
+<img src="https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics/grumpy-cat-small.png" width="250px">
+
+<table style="margin-left:100px;">
+
+    <tr>
+        <th>Code</th>
+        <th>Output</th>
+</tr>
+<tr>
+    <td style="padding-left:50px;">
+        <pre>&lt;li&gt;
+</pre>
```

Comment:
:hammer_and_wrench:  Bitte diese `<table>` anpassen mit den richtige HTML Format ^-^
```suggestion
        <pre>&lt;li&gt;</pre>
```